src/hpc.qmd

```diff
@@ -1,73 +1,432 @@
 ---
 title: "HPC"
+execute:
+  eval: false
 ---
 
 # What is HPC ?
-- Description of HPC, what it is and what it is used for.
+High performance computing (HPC) is the practice of aggregating ("clustering") computing resources that work in parallel to gain performance greater than that of a single workstation and process massive multi-dimensional data sets.
+
+A standard computing system solves problems primarily by using serial computing—it divides the workload into a sequence of tasks and then runs the tasks one after the other on the same processor. On the other hand, HPC uses parallel processing, which divides the workload into tasks that run simultaneously on multiple processors.
```

Comment:
computing-it ça veut dire computing unit ?

Reply:
Non, c'est un trait de séparation, un `:` serait plus clair. Je propose une reformulation : 

```suggestion
Computing system solves problems by dividing the workload into a subset of tasks. In a serial/sequential approach, tasks are executed one after the other on the same processor. In a parallel approach, the task are executed simultaneously/concurrently on mulitple processors.
```